### DISCUSSION TITLE
Suggestion of a column to Delta Compare Size vs just # of differences

### DISCUSSION THREAD

**Discussion by sltwo** [WinMerge/winmerge#2382] [Ideas]
Currently supported is the number of differences but it would be good to see the total size of the changes because in cases where we have a whole file content change which would show up at 1 change, to be able to see the size of the change.  When looking through changes this could be very helpful.   

![image](https://github.com/user-attachments/assets/e7f270cf-c439-452b-b369-eb307a011436)

**Comment by sdottaka** [comment]
Although it's a little different from what you suggest, it's possible to show the difference in file sizes. See the video at the URL below.
https://github.com/WinMerge/winmerge/issues/1520#issuecomment-1278287506

**Reply by sltwo** [reply]
Is this on the folder view or someplace else?  I see the old size and the new size for me to sort it could be very useful to sort on the amount of change.

**Reply by sdottaka** [reply]
The columns are displayed in the Folder compare window.

See the following gif for more details:

https://user-images.githubusercontent.com/98126/195730055-e95a3e98-746b-47c7-aeb0-85540295325e.gif
